monitor/src/main/java/com/iluwatar/monitor/Main.java

```diff
@@ -0,0 +1,55 @@
+package com.iluwatar.monitor;
+
+import java.util.*;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class Main {
+
+    public static void main(String[] args) {
+        AtomicReference<Bank> bank = new AtomicReference<>();
```

Comment:
You can use local variable type inference `var` in many places, instead of explicitly specifying the type

Reply:
i have changed it .